.github/workflows/checks.yml

```
@@ -201,14 +201,20 @@ jobs:
           node-version-file: .nvmrc
       - name: Install dependencies
         run: npm clean-install
-      - name: Transpile to CommonJS
-        run: npm run transpile
+      - name: Simulate publish
+        run: |
+          npm publish --dry-run
+          npm pack
       - name: Compute checksum
-        run: shasum index.cjs testing.cjs | tee checksums.txt
+        run: shasum -- *.tgz | tee checksums.txt
       - name: Reset to a clean state
-        run: npm run clean
-      - name: Transpile to CommonJS again
-        run: npm run transpile
+        run: |
+          npm run clean
+          rm -f -- *.tgz
```

Files must exist, so `-f` is unnecessary here

```suggestion
          rm -- *.tgz
```